### DISCUSSION TITLE
Polygon Full Node vs Archive Node

### DISCUSSION THREAD

**Discussion by choilive** [erigontech/erigon#6691] [Q&A]
Does erigon work for setting up a Polygon Full node? Both the documentation and polygon's website only refers to archive nodes so wondering if this is possible. If so- any rough estimates on that amount of disk space I would need?

**Comment by AskAlexSharov** [comment]
https://github.com/erigontech/erigon?tab=readme-ov-file#erigon3-datadir-size
